### ISSUE TITLE
Circular imports do not build

### ISSUE THREAD

**Issue by hazae41** [alephjs/aleph.js#89]
Hey, when building a project where two files import each other, the build fails

pages/index.tsx

```typescript
import { Component } from "../components/component.tsx"

export const App = () => {
  // ...
}
```

components/component.tsx

```typescript
import { App } from "../pages/index.tsx"

export const Component = () => {
  // ...
}
```

We get the following error

> ERROR TypeError: Cannot resolve module "file:///workspace/.aleph/production.es2015/pages/index.16b03cd86.js" from "file:///workspace/.aleph/production.es2015/components/component.e9b0a1ba3.js".

**Comment by shadowtime2000** [comment]
This will most likely fix with the new compiler #28 but I don't know for sure. Would circular imports work with something like NextJS tho?

**Comment by hazae41** [comment]
The bug still happens in 3.0 beta 8 (Deno 1.7.5)

Same error
